diesel/src/pg/expression/array_comparison.rs

```diff
@@ -145,8 +145,8 @@ where
     }
 }
 
-impl<ST, S, F, W, O, L, Of, G, FU> AsArrayExpression<ST>
-    for SelectStatement<S, F, W, O, L, Of, G, FU>
+impl<ST, F, S, D, W, O, LOf, G, H, LC> AsArrayExpression<ST>
```

Comment:
This occurrence seemed to be entirely incorrect with its ordering. Have amended but can revert if this was intentional.